test/literate_tabbed.litcoffee

```diff
@@ -69,13 +70,15 @@ if true
 
 	test "should ignore code blocks inside HTML", ->
 		eq executed, false
+		testsCount++
 
 ---
 
-*   A list item with a code block:
+*   A list item followed by a code block:
 
-		test "basic literate CoffeeScript parsing", ->
-			ok yes
+	test "basic literate CoffeeScript parsing", ->
```

Comment:
Custom rules are fine. As long as they are rendered correctly by most popular tools.

Imagine the security risk behind this.  A user see something on GitHub like this:

<hr>

    fetch_me_cute_cats()

* Don't worry

    because_this_is_not_code_and_will_totally_not_destroy_your_computer_or_anything()

<hr>

But literate Coffee sees it like this:

<hr>

    fetch_me_cute_cats()

* Don't worry

      because_this_is_not_code_and_will_totally_not_destroy_your_computer_or_anything()

<hr>

And the user happily execute it because why wouldn't trust GitHub's render?
The next day the user buy a new computer just so that he could submit an issue filled with anger here.

A little too dramatic maybe, but you get the idea.